### PR TITLE
Text editor

### PR DESCRIPTION
**Pull request by Sancharisingh**
Closes: #977 

- **Title:** Text Editor
- **Name:** Sanchari singh
- **Idenitfy yourself:** GSSOC ext and Hacktoberfest

### Describe the add-ons or changes you've made 📃
I added a project titled Text Editor in the React-JS Projects directory, within the Intermediate section. This Text Editor project brings essential text formatting capabilities, allowing users to create and edit content with various styling options.

### Screenshots 📷
Screenshot 2024-10-24 at 5 36 48 PM

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### CODE REVIEW

**Comment by Zedoman** [comment]
@Sancharisingh please provide a working video

**Comment by Sancharisingh** [comment]
> @Sancharisingh please provide a working video

https://youtu.be/PXinp04QQcg

File: React-JS-Projects/Intermediate/Text-Editor/Screenshot.webp
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Intermediate/Text-Editor/Screenshot.webp:1]: File name in small letters

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Enlist your project in react Readme file

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**